tests_e2e/test_suites/no_outbound_connections.yml

```diff
@@ -16,5 +16,5 @@ tests:
   - "bvts/vm_access.py"
   - "no_outbound_connections/check_fallback_to_hgap.py"
 images: "random(endorsed)"
-template: "no_outbound_connections/nsg_template.py"
```

Comment:
renamed 'nsg_template.py' to 'template.py' for consistency with the agent_not_provisioned suite